lib/Mojolicious/Command/Author/generate/secret.pm

```diff
@@ -0,0 +1,85 @@
+package Mojolicious::Command::Author::generate::secret;
```

Comment:
Looks like the command does not have tests, like other generators do. Also wondering if we really do want a separate config file for secrets, if the command should have an option to automatically generate and rotate in a new secret.

Reply:
Tests have been added for `mojo generate secret`